packages/client/src/sw/install.test.ts

```
@@ -0,0 +1,186 @@
+
+  addEventListener<K extends keyof ServiceWorkerEventMap>(
+    type: K,
+    listener: (event: ServiceWorkerEventMap[K]) => void
+  ): void {
+    this.listeners.set(type, listener as (event: never) => void);
+  }
+
+  dispatch<K extends keyof ServiceWorkerEventMap>(type: K, event: ServiceWorkerEventMap[K]): void {
+    const listener = this.listeners.get(type) as
+      | ((event: ServiceWorkerEventMap[K]) => void)
+      | undefined;
```

ESLint

[error] Replace `|·((event:·ServiceWorkerEventMap[K])·=>·void)⏎·····` with `((event:·ServiceWorkerEventMap[K])·=>·void)`

(prettier/prettier)